qa/workunits/fs/misc/mkpool_layout_vxattrs.sh

```diff
@@ -9,7 +9,5 @@ setfattr -n ceph.file.layout.pool -v foo.$$ foo.$$
 
 # cleanup
 rm foo.$$
-ceph fs rm_data_pool cephfs foo.$$
-ceph osd pool rm foo.$$ foo.$$ --yes-i-really-really-mean-it
```

Comment:
after flushing the journal, isn't there a way to check if the purge queue is empty to eventually decide on removing the data pool ?

Reply:
Sure, we could add a manual check to look at perf counters but how can we expect users/admins to do that? Until we have a proper interface, I don't see a point adding this to the tests.